.github/workflows/ci.yml

```diff
@@ -8,9 +8,21 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stack-yaml:
+          - stack.yaml
+          - stack-lts-18.0.yaml
+          - stack-lts-18.25.yaml
```

Comment:
While these exercise the persistent thing, they're both the same GHC version. We like to use the matrix to hit various GHCs -- which should also exercise the persistent thing.

```
GHC 8.10  => lts-18.26
GHC 8.8   => lts-16.31
GHC 8.6   => lts-14.27
GHC 8.4   => lts-12.26
GHC 8.2   => lts-11.22
GHC 8.0   => lts-9.21
GHC 7.10  => lts-6.35
GHC 7.8   => lts-2.22
```

I typically do 12, 14, and 16. But if the older ones aren't easy to get compiling, just 16 in addition to the default is OK with me. (I also don't typically duplicate the current `stack.yaml`, so no 18 necessary.)